data/bookkeeping/block.go

```diff
@@ -539,6 +560,31 @@ func (block Block) paysetCommit(t config.PaysetCommitType) (crypto.Digest, error
 	}
 }
 
+func (block Block) paysetCommitSHA256() (crypto.Digest, error) {
+	params, ok := config.Consensus[block.CurrentProtocol]
```

Comment:
I think that this condition is already checked in the caller function.

Reply:
I still need the params to verify that `EnableSHA256TxnRootHeader` is on, just ignoring the `ok` return value seems like bad practice.

Reply:
Actually I think I'll just move the `EnableSHA256TxnRootHeader` check to the caller as well instead